content/docs/components/hyperparameter-tuning/experiment.md

```diff
@@ -0,0 +1,477 @@
+
+For an overview of the concepts involved, read the [introduction to 
+Katib](/docs/components/hyperparameter-tuning/overview/).
+
+<a id="docker-image"></a>
+## Packaging your training code in a container image
+
+Katib and Kubeflow are Kubernetes-based systems. To use Katib, you must package
+your training code in a Docker container image and make the image available
+in a registry. See the [Docker
+documentation](https://docs.docker.com/develop/develop-images/baseimages/) and
+the [Kubernetes 
+documentation](https://kubernetes.io/docs/concepts/containers/images/).
+
+## Configuring the experiment
+ 
+To create a hyperparameter tuning or NAS experiment in Katib, you define the
+experiment in a YAML configuration file. The YAML file defines the range of 
+potential values (the search space) for the paramaters that you want to 
+optimize, the objective metric to use when determining optimal values, the 
+search algorithm to use during optimization, and other configurations.
+
+See the [YAML file for the random algorithm 
+example](https://github.com/kubeflow/katib/blob/master/examples/v1alpha3/random-example.yaml).
+
+The list below describes the fields in the YAML file for an experiment. The
+Katib UI offers the corresponding fields. You can choose to configure and run
+the experiment from the UI or from the command line.
+
+### Configuration spec
+
+These are the fields in the experiment configuration spec:
+
+* **parameters**: The range of the hyperparameters or other parameters that you 
+  want to tune for your ML model. The parameters define the *search space*,
+  also known as the *feasible set* or the *solution space*.
+  In this section of the spec, you define the name and the distribution 
+  (discrete or continuous) of every hyperparameter that you need to search.
+  For example, you may provide a minimum and maximum value or a list
+  of allowed values for each hyperparameter.
+  Katib generates hyperparameter combinations in the range based on the
+  hyperparameter tuning algorithm that you specify. See the [`ParameterSpec` 
+  type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/experiments/v1alpha3/experiment_types.go#L142-L163).
+
+
+* **objective**: The metric that you want to optimize. 
+  The objective metric is also called the *target variable*. 
+  A common metric is the model's accuracy in the validation pass of the training
+  job (*validation-accuracy*). You also specify whether you want Katib to 
+  maximize or minimize the metric.
+  Katib uses the `objectiveMetricName` and `additionalMetricNames` to monitor
+  how the hyperparameters work with the model. 
+  Katib records the value of the best `objectiveMetricName` metric (maximized 
+  or minimized based on `type`) and the corresponding hyperparameter set
+  in `Experiment.status`. If the `objectiveMetricName` metric for a set of
+  hyperparameters exceeds the `goal`, Katib stops trying more hyperparameter 
```

Comment:
s/exceeds/reaches? Since the objective can be either maximize or minimize.

Reply:
Good point. Done.